samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/serializers.dart

```diff
@@ -75,12 +73,10 @@ part 'serializers.g.dart';
   ArrayTest,
   Capitalization,
   Cat,
-  CatAllOf,$CatAllOf,
```

Comment:
looking at the dart samples in 
https://github.com/OpenAPITools/openapi-generator/tree/inline-schema-improve3/samples/openapi3/client/petstore/dart-dio

I see that [the inline model files still remain](https://github.com/OpenAPITools/openapi-generator/blob/inline-schema-improve3/samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/model/cat_all_of.dart) but  their usage is removed

what might be causing this ?

Reply:
https://github.com/OpenAPITools/openapi-generator/blob/inline-schema-improve3/samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/model/cat_all_of.dart is left over from previous code generation.

I can confirm manually removing https://github.com/OpenAPITools/openapi-generator/blob/inline-schema-improve3/samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/src/model/cat_all_of.dart and regenerating dart samples will recreate this file, which is the expected behaviour as part of this change.